buddyboss/buddyboss-platform
# Order for All results tab for members fixed

### Changes proposed in this Pull Request:



Fixes #2427

### How to test the changes in this Pull Request:

1. Search a name with first name & last name
2. Hit Enter
3. See the result at All Results & Members tabs
4. See the ordering for the Searched name.

### Proof Screenshots or Video


https://www.loom.com/share/0ca93e8d7a4f4143bee4a26e962952a0
